bin/beaver-install

```diff
@@ -24,15 +24,13 @@ do
     esac
 done
 
-d="$(dirname "$0")"
-
 if test -r "Dockerfile${DIST:+.$DIST}"
 then
     # Build the docker image using .$DIST as suffix if present
-    "$d/beaver" docker build -f "Dockerfile${DIST:+.$DIST}" "$@" .
+    beaver docker build -f "Dockerfile${DIST:+.$DIST}" "$@" .
```

Comment:
Why do you want to require this (`beaver` being on the `PATH`)??? It's making something that works more generally to be more restrictive for which gain?

Reply:
Because it is already effectively required (beaver doesn't work when called via `submodules/bin/beaver-something`, I tried that) and this extra indirection just makes sources harder to read.

Reply:
I sometimes use it and it works for me. Maybe you hit a bug. I can see a point on simplifying things (in a way), but I'm not completely convinced on removing this possibility yet. When using it more locally is very useful to be able to use `beaver/bin/beaver` without touching the `PATH`.